build-wheelhouse/action.yml

```diff
@@ -89,7 +81,8 @@ runs:
     - name: "Verify library is properly installed and get its version number"
       shell: bash
       run: |
-        version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(); print(__version__)" | tail -1 || exit 1 )
+        library_name=${{ inputs.library-name }}
+        version=$(python -c "import importlib.metadata as metadata; print(metadata.version('$library_name'))")
```

Comment:
This will only be allowed from Python 3.8 onwards. Maybe we should specify it in the description.

Reply:
You are right. Let me fix this by installing the `importlib-metadata` package if `inputs.python-version` is lower than Python 3.8. Also, I will update the version retrieval command to be compliant with these new changes.